SimPEG/regularization/pgi.py

```diff
@@ -94,21 +94,35 @@ class PGIsmallness(Smallness):
     (:math:`\boldsymbol{\Sigma}`) and proportion constants (:math:`\boldsymbol{\gamma}`)
     defining the GMM. And let :math:`\mathbf{z}^\ast` define an membership array that
     extracts the GMM parameters for the most representative rock unit within each active cell
-    in the :class:`RegularizationMesh`. The regularization function (objective function) for
-    ``PGIsmallness`` is given by:
+    in the :class:`RegularizationMesh`.
+
+    When the ``approx_eval`` property is ``True``, we assume the physical property distributions of each geologic units
+    are distinct (no significant overlap of their respective physical properties distribution). The GMM probability
+    density value at any each point of the physical property space can then be approximated by the locally dominant
+    Gaussian distribution. In this case, the PGI regularization function (objective function) can be expressed as a
+    least-square:
```

Comment:
I think these lines belong to another PR. Can you revert them back to the original state since they aren't related to the mag simulation?

Reply:
The commits with these changes (and the lines below) are merged into SimPEG `main` -- if you sync your fork the diff should go away